9_collaborator_data.yml

```diff
@@ -253,3 +254,28 @@ targets:
       network_ind = '1_network/out/network.rds.ind',
       out_file = target_name)
 
+  #### Reservoirs ####
+
+  res_outflow_ids:
+    command: list(
+      Cannonsville = I('01425000'),
+      Pepacton = I(c('01417000', '01417500')))
+
+  # Find sites with lots of flow and/or temp data on inflows to each reservoir.
+  # This algorithm uses NLDI and hence currently only locates NWIS sites, overlooking
+  # any other relevant sites in WQP
+  res_inflow_ids:
+    command: find_inout_obs_sites(
+      res_outflow_ids = res_outflow_ids,
+      flow_ind = '2_observations/in/daily_flow.rds.ind',
+      temp_ind = '2_observations/out/all_drb_temp_obs.rds.ind',
+      min_obs_flow = I(1000),
+      min_obs_temp = I(50),
```

Comment:
Is the difference in data requirements between flow and temp because many NGWOS temp sites just recently came online?

Reply:
Hmm, I'd like your additional input on this. As far as I can tell, temperature data are quite limited for inflows. Yes, I set `min_obs_temp=50` because that seemed necessary to get any sites -- when I tried `min_obs_temp=1000`, I got 0 sites back. Here are the temperature data plots generated by the code in this PR for the sites discovered by this NLDI algorithm:

![can_io_temp](https://user-images.githubusercontent.com/12039957/110379148-711f8080-8024-11eb-82db-eae3e77af91c.png)

![pep_io_temp](https://user-images.githubusercontent.com/12039957/110379170-7a105200-8024-11eb-9e49-5450d414449e.png)

So my NLDI algorithm helped me discover that inflow sites 01423000 (Cannonsville) and 01413088 (Pepacton) had some temperature data for inflows (red points). And site 01423000 looks like it's probably an NGWOS site (starts in ~2019). But is it right that Pepacton doesn't have any NGWOS inflow sites? If it does, my algorithm didn't find them.

Reply:
Alison and I did some investigating and had some conversation in Teams, so putting Alison's conclusion here: 

> To sum up : I think 01413088 is OK because it's WQP, not NWIS data. I think 01415460 just needs a more recent temp data pull (if the last pull really was 2020-01-15?). And I can't explain why 01413500 and 01415000 don't have more data in all_drb_temp_obs.rds. Hmm.

Long story short, it looks like prior to 3 months ago, provisional data were being filtered out in the national pipeline. This originally wasn't an issue in this pipeline because I was pulling NGWOS data directly in this pipeline using a site list. Once the national pull was more current, I figured, I can just get NGWOS data from there! I last updated the DRB pipeline with national data in November. But in December, I repulled NWIS data to get min/max values, and [noticed and fixed](https://github.com/USGS-R/2wp-temp-observations/commit/9c6e1b4476079d97a32606fbeaf4c9dcb70a221b) the provisional data issue, but did not pipe the new national data over to the DRB pipeline. 

The solution is to go grab the [latest national data](https://drive.google.com/drive/folders/1g0fOcRbQNsG3G9OXe_FZ4YhYXqrHT_vh), and place in or overwrite `2_observations/in/daily_temperatures.rds` and rerun the DRB pipeline. That might make sense for you to do, Alison, since you have this PR open, but I am happy to do it if that's easier.

Reply:
![image](https://user-images.githubusercontent.com/15788176/110626928-f8750d00-8166-11eb-9eeb-6bd9bd975f18.png)

Reply:
Finally, a note about the weirdness between NWISuv and NWISdv for these newer NGWOS sites. Sites `01415000` and `01415460` aren't in dv, and you get slightly different answers pulling `01413500`.

```
# dv
missing_dv <- dataRetrieval::readNWISdv(c('01415460', '01413500','01415000'), parameterCd = '00010') 
summary_dv <- group_by(missing_dv, site_no) %>% 
  summarize(n = n(), n_p = sum(X_00010_00003_cd %in% 'P'))
# uv
missing_uv <- dataRetrieval::readNWISuv(c('01415460', '01413500','01415000'), parameterCd = '00010') 
test_uv <- mutate(missing_uv, date = as.Date(dateTime)) %>%
  group_by(site_no, date) %>%
  slice_max(X_00010_00000, n = 1, with_ties = FALSE) %>%
  group_by(site_no) %>%
  summarize(n = n(), n_p = sum(X_00010_00000_cd %in% 'P'))
```
dv results:
![image](https://user-images.githubusercontent.com/15788176/110631484-2f99ed00-816c-11eb-8794-0feaa35aeba4.png)

uv results:
![image](https://user-images.githubusercontent.com/15788176/110631383-12651e80-816c-11eb-9852-aeb4e39ffe0e.png)

Reply:
Looks like start date is off, too?

dv: 
![image](https://user-images.githubusercontent.com/15788176/110645716-d639ba00-817b-11eb-94eb-2db344635846.png)

uv:
![image](https://user-images.githubusercontent.com/15788176/110645635-c3bf8080-817b-11eb-8ad0-5fcfa365c366.png)

Reply:
Makes sense that 3/10 wouldn't be in dv yet.

Reply:
Probably the first day wasn't a complete day. And maybe there were 3 other incomplete days (just a few missing hours here or there) elsewhere in the record.